components/Alert.vue

```diff
@@ -17,8 +17,8 @@
 <script>
 export default {
   props: {
-    buttonText: String,
-    buttonUrl: String,
```

Comment:
It's always good to define default types in case something is misconfigured

Reply:
FWIW, My props rule is _it must be either required or either having a default value_.